RadarSDK/RadarReplayBuffer.m

```diff
@@ -57,6 +62,19 @@ - (void)writeNewReplayToBuffer:(NSMutableDictionary *)replayParams {
  */
 - (void)clearBuffer {
     [mutableReplayBuffer removeAllObjects];
+
+    // remove persisted replays
+    [[NSUserDefaults standardUserDefaults] removeObjectForKey:@"radar-replays"];
+}
+
+- (void)loadReplaysFromPersistentStore {
+    [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug message:@"Loading replays from persistent store"];
+    NSData *replaysData = [[NSUserDefaults standardUserDefaults] objectForKey:@"radar-replays"];
+    if (replaysData) {
+        NSArray *replays = [NSKeyedUnarchiver unarchiveObjectWithData:replaysData];
+        [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug message:[NSString stringWithFormat:@"Loaded replays with length %lu", (unsigned long)[replays count]]];
+        mutableReplayBuffer = [NSMutableArray arrayWithArray:replays];
+    }
```

Comment:
is it worth having a log line for "No replays found from persistent store" if `replaysData` is empty?

Reply:
can be inferred from lack of the "loaded replays with..."